less/navigation.less

```diff
@@ -1 +1,39 @@
-// Navigation Styles here
\ No newline at end of file
+// Navigation Styles here
+
+// MIXINS
+.flexbox(@flex-dir: row, @just-cont: flex-start, @align-items: stretch) {
```

Comment:
You can remove this mixin because you have imported the flexbox mixin from `mixins.less` already 👍